tests/Unit/ApparentHorizons/Test_FastFlow.cpp

```diff
@@ -288,7 +288,15 @@ void test_kerr(FastFlow::Flow::type type_of_flow, const double mass,
 SPECTRE_TEST_CASE("Unit.ApparentHorizons.FastFlowSchwarzschild",
                   "[Utilities][Unit]") {
   test_schwarzschild(FastFlow::FlowType::Fast, 100);
+}
+
+SPECTRE_TEST_CASE("Unit.ApparentHorizons.JacobiSchwarzschild",
+                  "[Utilities][Unit]") {
   test_schwarzschild(FastFlow::FlowType::Jacobi, 200);
+}
+
+SPECTRE_TEST_CASE("Unit.ApparentHorizons.CurvatureSchwarzschild",
+                  "[Utilities][Unit]") {
```

Comment:
For my understanding: we've generally been asked to reduce the number of test cases... what is the reason for splitting 1 into 3 here?

Reply:
single test times out